tools/actions/commit-queue.sh

```diff
@@ -33,6 +33,13 @@ git config --local user.name "Node.js GitHub Bot"
 
 for pr in "$@"; do
   gh pr view "$pr" --json labels --jq ".labels" > labels.json
+
+  # Skip if "blocked" label is present
+  if jq -e 'map(.name) | index("blocked")' < labels.json; then
+    echo "pr ${pr} skipped, 'blocked' label present"
+    continue
```

Comment:
We shouldn't do that, we should either add a comment to the PR and remove the `commit-queue` label, or we should filter out `blocked` PRs in the previous step: https://github.com/nodejs/node/blob/7f392d4a144ac4b2f4a42afffd05bf19c479fe31/.github/workflows/commit-queue.yml#L33-L53

The reason the current approach is problematic is that it's going to force the GHA to re-clone the full git history every 5 minutes, for no good reason.